tests/ui/offset-of/offset-of-enum.rs

```diff
@@ -9,5 +9,6 @@ enum Alpha {
 
 fn main() {
     offset_of!(Alpha::One, 0); //~ ERROR expected type, found variant `Alpha::One`
-    offset_of!(Alpha, Two.0); //~ ERROR no field `Two` on type `Alpha`
+    offset_of!(Alpha, One); //~ ERROR `One` is an enum variant; expected field at end of `offset_of`
+    offset_of!(Alpha, Two.0);
```

Comment:
Could there be also a test for `offset_of!(Alpha, Two.1)` and `Two.foo`? Also, one test for `offset_of!(Alpha, NoEx)` to test the error message for a non-existent variant.